assets/js/components/adminbar/AdminBarApp.js

```diff
@@ -75,7 +80,36 @@ export default function AdminBarApp() {
 						mdc-layout-grid__cell--align-middle
 					">
 						<div className="mdc-layout-grid__inner">
-							<AdminbarModules />
+							{ featureFlags.widgets.adminBar.enabled
+								? (
+									<Fragment>
+										{
+											( analyticsModuleActive && analyticsModuleConnected )
+												? (
+													<Fragment>
+														<AdminBarUniqueVisitors />
+														<AdminBarSessions />
+													</Fragment>
+												)
+												: (
+													<div className="
+														mdc-layout-grid__cell
+														mdc-layout-grid__cell--span-6-desktop
+														mdc-layout-grid__cell--span-4-tablet
+													">
+														<AnalyticsInactiveCTA />
+													</div>
+												)
+										}
+
+										<AdminBarImpressions />
+										<AdminBarClicks />
+									</Fragment>
+								)
+								: (
+									<AdminbarModules />
+								)
```

Comment:
According to the ACs, the `AdminbarModules` would still render regardless of the feature flag value. Similar to the main dashboard – if the flag is enabled, the new widget version would render above _in addition_ to the legacy version below it.

Reply:
Updated @aaemnnosttv